.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no <!-- please update CHANGELOG.md file -->
+| Deprecations? | yes/no <!-- please update CHANGELOG.md file -->
+| Issues        | Fix #... <!-- prefix each issue number with "Fix #", no need to create an issue if none exists, explain below instead -->
+| License       | MIT
+
+<!--
+Replace this notice by a description of your feature/bugfix.
+This will help reviewers and should be a good start for the documentation.
+
+Additionally (see https://symfony.com/releases):
+ - Always add tests and ensure they pass.
+ - Features and deprecations must be submitted against the latest branch.
+ - For new features, provide some code snippets to help understand usage.
+ - Changelog entry should follow https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry
+ - Never break backward compatibility (see https://symfony.com/bc).
```

Comment:
the link to `https://symfony.com/bc` should be removed as this is about allowed changes in *PHP* code, which won't help in this repo.

Reply:
Done